features/admin/resetting_password.feature

```diff
@@ -0,0 +1,35 @@
+@administrator_login
+Feature: Resetting an administrator's password
+    In order to login to my administrator account when I forget my password
+    As an not logged in Administrator
+    I need to be able to reset my password
+
+    Background:
+        Given there is an administrator "sylius@example.com" identified by "sylius"
+
+    @todo
+    Scenario: Resetting an administrator password
+        When I want to reset password
+        And I specify email as "sylius@example.com"
+        And I reset it
+        Then I should be notified that email with reset instruction has been sent
+        And an email with reset token should be sent to "sylius@example.com"
```

Comment:
the administrator should not be aware of the "reset token", he only gets an email with a link and instructions on how to change the password
`And an email with instructions on how to reset the password should be sent to "sylius@example.com"